### PR TITLE
posthog-inspector-docs

### PR DESCRIPTION
**Pull request by bjornj12**
Docs for setting up the posthog plugin for avo inspector

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…t-inspector-to-posthog**](https://docs-inlyzu8sf.avo-dev.app/workspace/connect-inspector-to-posthog) |
|:-:|
|<a href="https://docs-inlyzu8sf.avo-dev.app/workspace/connect-inspector-to-posthog">Screenshot of /workspace/connect-inspector-to-posthog</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/p-s0Ybp3gDEW32Z2XYvC42P8Go4MXTQKGOTZsvuI3LVLM0wb5bR99BDe0Ro7osfIYLxlWsPu4eUGbZ7sXQtiBA.png">(view full size)</a> |







Commit 3a104fafcd73d2cfbb86857408175a17afe64425 (https://docs-inlyzu8sf.avo-dev.app).

File: .spelling [hunk 1]
```diff
@@ -386,6 +386,8 @@ csharp#destination-interface-example
 mParticle
 setUserProperties
 PII
+Rudderstack
```
Comment (logason) [inline .spelling:389]:
```suggestion
RudderStack
```

File: components/Navigation.tsx [hunk 2]
```diff
@@ -337,6 +337,11 @@ const navigation: navigationItem[] = [
         title: 'Inspector RudderStack integration',
         path: '/workspace/connect-inspector-to-rudderstack',
       },
+      {
+        type: 'route',
+        title: 'Inspector RudderStack integration',
```
Comment (logason) [inline components/Navigation.tsx:342]:
```suggestion
        title: 'Inspector PostHog integration',
```

File: pages/implementation/devs-101.mdx [hunk 3]
```diff
@@ -105,7 +105,8 @@ You can pipe your analytics data into the Inspector API via your CDPs or analyti
 - Segment: <Link passHref href="/workspace/connect-inspector-to-segment"><a>Set Inspector up via Segment Functions</a></Link>
 - Mixpanel: Coming soon.
 - mParticle: Coming soon.
-- RudderStack: Coming soon.
+- RudderStack: <Link passHref href="/workspace/connect-inspector-to-rudderstack"><a>Set Inspector up via RudderStack Functions</a></Link>
+- PostHog: <Link passHref href="/workspace/connect-inspector-to-posthog"><a>Set Inspector up via PostHog Plugin</a></Link>
```
Comment (logason) [inline pages/implementation/devs-101.mdx:109]: Should we move the coming soon destinations below?

File: pages/workspace/connect-inspector-to-posthog.mdx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+title: 'Send data from PostHog to Avo Inspector'
+---
+
+import Link from '../../components/Link';
+
+## Use Avo Inspector as a PostHog destination
+
+If you are already sending your data to PostHog you can stream it to Avo Inspector without adding any code to your codebase.
+
+PostHog has an official plugin maintained by PostHog that sends data directly to the Avo Inspector. If you are using Posthog you can set it up within a few minutes.
```
Comment (logason) [inline pages/workspace/connect-inspector-to-posthog.mdx:12]:
```suggestion
PostHog has an official plugin maintained by PostHog that sends data directly to the Avo Inspector. If you are using PostHog you can set it up within a few minutes.
```

File: pages/workspace/connect-inspector-to-posthog.mdx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+> The Plugin is a side effect of the Avo Inspector analyzing your events and keeping your PostHog events stream unchanged.
+
+After connecting the PostHog plugin to a source the event schemas of all events seen in PostHog will automatically be sent to Avo Inspector and you'll be able to monitor the quality of your tracking implementation without any changes in your code!
+
+> Remember that production data takes up to 2 hours to appear in the Inspector dashboard, while development data is available without any delay
```
Comment (logason) [inline pages/workspace/connect-inspector-to-posthog.mdx:18]: Should we also highlight that Inspector only receives the event schemas, and not the actual data?

File: pages/workspace/connect-inspector-to-posthog.mdx [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+<center style={{marginBottom: 16 + "px"}}>
+  <img
+    width="100%"
+    src={require('../../images/workspace/nav-inspector-sources-highlighted.png')}
+    alt="Navigation to the Inspector sources screen in Avo"
+  />
+</center>
+
+If you already have the source, pick it. If you don’t have the source corresponding to your app, <Link passHref href="/workspace/inspector#configuring-sources"><a>add a new one</a></Link>.
```
Comment (logason) [inline pages/workspace/connect-inspector-to-posthog.mdx:35]:
```suggestion
If you already have the source, open it in the dashboard. If you don’t have the source corresponding to your app, <Link passHref href="/workspace/inspector#configuring-sources"><a>add a new one</a></Link>.
```

File: pages/workspace/connect-inspector-to-posthog.mdx [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+<center style={{marginBottom: 16 + "px"}}>
+  <img
+    src={require('../../images/add-source.png')}
+    alt="Image of the Add Source button in Avo"
+  />
+</center>
+
+In the source details, go to "Inspector Setup" and there you’ll see options to integrate Inspector. Pick PostHog to see the integration code snippets.
```
Comment (logason) [inline pages/workspace/connect-inspector-to-posthog.mdx:44]:
```suggestion
In the source details, go to "Inspector Setup" and there you’ll see options to integrate Inspector. Pick PostHog to see the integration guide.
```

File: pages/workspace/connect-inspector-to-posthog.mdx [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+<center style={{marginBottom: 16 + "px"}}>
+  <img
+    src={require('../../images/guide-posthog-to-inspector/insert-api-key.png')}
+    alt="PostHog insert avo API key"
+  />
+</center>
+
+#### Step 3. Test your PostHog plugin integration on Dev.
+
+> You can test the production PostHog Plugin in the same way, but keep in mind that the production data shows with a ~2 hours delay in the Inspector dashboard, while development data is shown without delay
```
Comment (logason) [inline pages/workspace/connect-inspector-to-posthog.mdx:85]:
```suggestion
> You can test the production PostHog Plugin in the same way, but keep in mind that the production data shows with a ~2 hours delay in the Inspector dashboard, while development and staging data is shown without delay
```

File: pages/workspace/connect-inspector-to-posthog.mdx [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+<center style={{marginBottom: 16 + "px"}}>
+  <img
+    height="500"
+    src={require('../../images/guide-segment-to-inspector/source-in-inspector.png')}
+    alt="Source overview in Inspector"
+  />
+</center>
+
+This means that the PostHog plugin is working.
```
Comment (logason) [inline pages/workspace/connect-inspector-to-posthog.mdx:116]:
```suggestion
This means that the PostHog plugin is working 🎉
```

**Review by logason — APPROVED** [review]
LGTM!

**Pull request merged**